src/DataTable/RowStatus.jsx

```diff
@@ -1,17 +1,23 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
+import DataTableContext from './TableContext';
 
-const RowStatus = ({ className, pageSize, itemCount }) => (
-  <div className={className}>Showing {pageSize} of {itemCount}</div>
-);
+const RowStatus = ({ className, itemCount }) => {
+  const instance = useContext(DataTableContext);
```

Comment:
Leftover from before? Might be clearer to just pull out page and rows

Reply:
Oh, it's probably left over from debugging; will fix.